.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "bower_components"]
```

Comment:
As i mentioned, I'd prefer a build step from another branch that built `master`, and the bower components would be checked in there. However, due to how github pages works, the submodule is for my sanity right now so that there is always a working demo.